editor/syncdocument.cpp

```diff
@@ -193,7 +193,6 @@ bool SyncDocument::save(const QString &fileName)
 		return false;
 	}
 	QTextStream streamFileOut(&file);
-	streamFileOut.setCodec("UTF-8");
```

Comment:
This subtly breaks the Qt-5 build, where the default codec depends on the locale. In Qt6, UTF-8 is the default, but not Qt5...

Reply:
I think the right thing to do is instead this:

```C++
#if QT_VERSION < QT_VERSION_CHECK(6, 0, 0)
	streamFileOut.setCodec("UTF-8");
#endif
```